docs/en/observability/aws-serverless-transform.asciidoc

```diff
@@ -0,0 +1,412 @@
+:aws: AWS
+
+[[aws-serverless-transform]]
+= Transform
+

```

Comment:
Can we put the various transform topics as links in the top as well (like table of content)? Right now the user has to scan the entire page to know what kind of data transformation features are available on the page and its a long page so information is not easy to find.

Reply:
e.g.
You can transform or enrich data from {aws} as it is parsed by the Elastic Serverless Forwarder. This includes examples such as using tags and filters to organise your data and exclude specific messages, automatically discovering and collecting JSON content, and managing multiline messages effectively.